lib/ex_unit_fixtures/auto_import.ex

```diff
@@ -0,0 +1,81 @@
+defmodule ExUnitFixtures.AutoImport do
+  @moduledoc """
+  A mechanism for automatically importing fixture modules into the current file.
+
+  When you `use ExUnitFixtures.AutoImport`, it will automatically lookup
+  `fixtures.exs` files in the current and parent directories, and import them
+  into the current test file for use.
+
+  They will be imported in descending file heirarchy order, so
+  `test/imp/fixtures.exs` can refer to & override fixtures in
+  `test/fixtures.exs`, but not the other way round.
```

Comment:
This could be documented better - could use the py.test documentation for inspiration, since it is quite good (and very similar)